bigchaindb/common/schema.py

```diff
@@ -0,0 +1,166 @@
+"""
+Schema definitions for BigchainDB data structure.
+
+Schema definitons are defined here in classes which are then translated
+into Json Schema (http://json-schema.org/). The classes are used for
+readability and to make documenting easier.
+"""
+
+
+def nullable(data):
+    """ Describe optional data """
+    return {"anyOf": [data, {"type": "null"}]}
+
+
+class SchemaObject(object):
```

Comment:
You can drop `(object)` since in Py3 all classes are "new style" classes by default. The examples in the [official documentation](https://docs.python.org/3/tutorial/classes.html#class-definition-syntax) don't inherit from `object`.